en/docs/troubleshooting/removing-unused-tokens-from-the-database.md

```diff
@@ -50,59 +50,62 @@ retain_access_tokens_for_auditing = true
 
 ### Using stored procedures for token cleanup
 
-Alternatively, you can also use the stored procedures provided below, to run a token cleanup task periodically to remove the old and invalid tokens.
+Alternatively, you can also use the stored procedures provided below, to run a token cleanup task periodically. This will remove the old and invalid tokens, sessions and auth codes, which cannot be cleaned by the products inbuilt cleanup process.
```

Comment:
Do we need to mention about session etc, because there are separate scripts to do registry and session cleanup.